### DISCUSSION TITLE
Weekly Sync 31st Jan 2022

### DISCUSSION THREAD

**Discussion by matdehaast** [interledger/rafiki#224] [General]
## Updates 

- Brandon
  - Working on WM balance withdrawal
  - Found some issues with the webhook implementation, going to consolidate it.
  - Aim for end of week webhooks PR and WM working.
- Sabine
   - Working with @adrianhopebailie and @AlexLakatos on the OPv2 spec.
- Alex
  - JD for Typescript for Interledger developer is live
  - Goal is for dev for Rafiki full time
- Damien
   - Still working on Coil/Franklin stuff.
- Matt
   - Working on [Rafiki Deployment Initial Plan](https://docs.google.com/document/d/1Dz5W5nZzt2q6BrpuI9RE0-5UQH6Q1dV-4xUx-EKCPng/edit#heading=h.qibwfqwe2qr)

## Review/Spec/TODO
- [ ] @sabinebertram  share Spec doc with @traviscrist @matdehaast 
- [ ] [TB upgrade](https://github.com/coilhq/tigerbeetle/pull/75) this week. Need to update all Rafiki

**Comment by A-Mozeak** [comment]
Hey there, I saw on a previous discussion that we can join the weekly calls. When/where can we join them? I'd love to keep up with the design and provide input if I can offer anything helpful.
